### PR TITLE
fix: removed blunt ids and changed of some

### LINKED ISSUE #706: fix: senseless names for ids in activity_login
**Summary:**
<TextView
            android:id="@+id/textView2"
            android:layout_width="wrap_content"
            android:layout_height="wrap_content"
            android:layout_marginStart="0dp"
            android:layout_marginLeft="0dp"
            android:layout_marginEnd="@dimen/marginBetweenWords"
            android:layout_marginRight="@dimen/marginBetweenWords"
            android:text="@string/don_t_have_an_account"
            android:textColor="@color/colorTextSecondary" />

       <TextView
            android:id="@+id/textView"
            android:layout_width="wrap_content"
            android:layout_height="wrap_content"
            android:text="@string/create_one"
            android:textColor="@color/colorAccent" />

These views have senseless ids and ids could be changed according to it's purpose.

### PR DESCRIPTION
**Pull request by Bhavnaharitsa**
Fixes: #706
Removed blunt ids and changed names of some ids

- [.] Apply the `AndroidStyle.xml` style template to your code in Android Studio.

- [.] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [.] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

**Comment by Bhavnaharitsa** [comment]
@luckyman20 Please review, this is a long made PR

File: mifospay/src/main/res/layout/activity_login.xml [hunk 1]
```diff
@@ -94,7 +93,7 @@
         app:layout_constraintStart_toStartOf="parent">
 
         <TextView
-            android:id="@+id/textView2"
+
```
Comment (luckyman20) [inline mifospay/src/main/res/layout/activity_login.xml]: Why did you add a new line?

File: mifospay/src/main/res/layout/activity_mobile_verification.xml [hunk 2]
```diff
@@ -6,19 +6,18 @@
     android:orientation="vertical">
 
     <LinearLayout
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
+        android:layout_width="match_parent"
+        android:layout_height="211dp"
```
Comment (luckyman20) [inline mifospay/src/main/res/layout/activity_mobile_verification.xml]: This doesn't seem to be a change related to this issue.

**Review by luckyman20 — COMMENTED** [review]
If the IDs aren't used anywhere then remove them altogether because I see that you have changed the ID names in XML files but haven't changed them in the Java files.

**Comment by Bhavnaharitsa** [comment]
@luckyman20 Okay, I'll see the usage and remove accordingly.

File: mifospay/src/main/res/layout/activity_settings.xml [hunk 3]
```diff
@@ -29,6 +29,7 @@
                 <TextView
                     android:layout_width="wrap_content"
                     android:layout_height="wrap_content"
+                    android:id="@+id/notification_setting"
```
Comment (rachittt) [inline mifospay/src/main/res/layout/activity_settings.xml:32]: this change was not required. kindly remove it

**Comment by rachittt** [comment]
as pointed out by @luckyman20, there are a few changes in the PR which are not relevant to the issue this PR is focusing on, also we can remove those IDs which are not being used programmatically. 

Please make the necessary changes. Thanks

**Comment by rchtgpt** [comment]
Closing this because of inactivity. Also, create different issues for different things to ensure modularity.
